### PR TITLE
add: `font-downloader`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Run pre-commit hooks.

**Comment by soumyatheman** [comment]
All the tests will pass, i ran the pre-commit.ci, and everythng was fine. Give it some time

**Comment by wizard-28** [comment]
> All the tests will pass, i ran the pre-commit.ci, and everythng was fine. Give it some time

The pre-commit CI doesn't run all the checks, like formatting. Run the pre-commit hooks locally on your machine and push.

**Comment by soumyatheman** [comment]
Any guide link for that? I mean how to do so....

**Comment by wizard-28** [comment]
It's in the README: https://github.com/pacstall/pacstall-programs#how-to-setup-the-environment-for-pacscript-development

**Comment by soumyatheman** [comment]
Do I need to do something else?

`````````````````
 soumyaxubuntu  soumyas-HP-Laptop-15-bs1xx  ../pacstall-programs  font-downloader  pre-commit
trim trailing whitespace.............................(no files to check)Skipped
fix end of files.....................................(no files to check)Skipped
check yaml...........................................(no files to check)Skipped
check for added large files..........................(no files to check)Skipped
file contents sorter.................................(no files to check)Skipped
mixed line ending....................................(no files to check)Skipped
Test shell scripts with shellcheck...................(no files to check)Skipped
Check shell style with shfmt.........................(no files to check)Skipped

**Comment by wizard-28** [comment]
To run `pre-commit` on specific files, use the following command from the project root:

```bash
pre-commit run --files packagelist packages/font-downloader/font-downloader.pacscript
```

Note that this only needs to be done once. `pre-commit` will handle this automatically in the future, assuming you've installed it using `pre-commit install`

File: packages/font-downloader/font-downloader.pacscript [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+gives="font-downloader"
+version="10.0.0"
+build_depends="cmake meson libhandy-1-dev gettext"
+# HACK: https://github.com/pacstall/pacstall/issues/727
+depends="libgtk-3-dev python3-gi"
+url="https://github.com/GustavoPeredo/Font-Downloader/archive/refs/tags/v${version}.zip"
+repology=("project: fontdownloader")
+description="Install fonts from online sources"
+hash="eeafd4ac9cb0d47fd0c1512e07805d0f7a639cdbbc688647249eaee8d1753e23"
+maintainer="সৌম্যদীপ <soumyadeepghosh2004@zohomail.in>"
```
Comment (wizard-28) [inline packages/font-downloader/font-downloader.pacscript:11]: By the way @soumyatheman, you have read the notice about automatic issue assignments right?

Comment (soumyatheman) [reply packages/font-downloader/font-downloader.pacscript:11]: Yes I read it. The package maintainers will get auto assigned.

Comment (wizard-28) [reply packages/font-downloader/font-downloader.pacscript:11]: No, you won't be auto-assigned if you don't use your GitHub username in the `maintainer` field. So, if you wish to be auto-assigned in the future, change your name from `সৌম্যদীপ` to `soumyatheman` in all the pacscripts.

**Review by wizard-28 — APPROVED** [review]
I interpreted you marking the conversation as resolved as you wanting to opt out of automatic issue assignments.

**Comment by soumyatheman** [comment]
Yess, because, whenever there'll be some issue, 90% cases, I'll also face that. Also, I  always have a close eye in the repo, So, problem in that also.

**Pull request merged**